### DISCUSSION TITLE
Multiple instances

### DISCUSSION THREAD

**Discussion by fkretlow** [ueberdosis/tiptap#1879] [General Questions]
I'm working on an app where I need to handle multiple small instances of the editor. I had a weird bug where all but the first editor ever created would gulp trailing whitespace on input, making it impossible to input spaces.

I found out that this was caused by destroying a previous instance of the editor *after* creating the new one. I can work around this by being really careful with the order in which creation and destruction are triggered, but it makes me wonder:

Does tiptap/ProseMirror even support working with multiple instances at the same time, or do instances share some internal objects so conflicts are likely to happen at some point down the road?

**Comment by philippkuehn** [comment]
Hmm, using multiple instances should be absolutely no problem. Is there any chance you could reproduce this in a codesandbox?

**Reply by fkretlow** [reply]
Sure, will do next week.

**Reply by fkretlow** [reply]
Apologies for the delay in getting back to this.

I can't seem to reproduce this in a minimal example. I guess there was something wrong with how I was handling attributes of Svelte component instances in the external handler. I'm afraid I don't have the time to investigate any further now.
